### PR TITLE
Gitpod Integration

### PR DESCRIPTION
**Pull request by Dylanb-dev**
You need to refresh the vs code plugin sometimes but everything else works.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/lolcookie/elm-spa-lamdera)
```
Comment (supermario) [inline README.md]: Should this reference this repo instead?

File: .gitpod.yml [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+tasks:
+  - init: yes | lamdera reset
+    command: sudo lamdera live
+ports:
+  - port: 8000
+    onOpen: open-preview
+
+vscode:
+  extensions:
+    - elmtooling.elm-ls-vscode
```
Comment (supermario) [inline .gitpod.yml:16]: Is there a way to configure the Elm path to point to the Lamdera binary? That would probably alleviate some issues down the line.

**Review by supermario — COMMENTED** [review]
Hey @lolcookie - thanks for working on this!

File: README.md (hunk 1 above)
Comment (Dylanb-dev) [reply README.md]: yep, updated

File: .gitpod.yml (hunk 2 above)
Comment (Dylanb-dev) [reply .gitpod.yml:16]: I added a .settings file to set this for the elm LS. Its breaking now though and I'm not sure why 😴

**Comment by Dylanb-dev** [comment]
DO NOT MERGE, NOT WORKING

**Comment by Dylanb-dev** [comment]
@supermario everything works fine on this version except it is using elm binary instead of lamdera for the LSP. Not sure if that is a mergable condition for this PR

**Comment by supermario** [comment]
@lolcookie nice!

I think it could be a rather frustrating experience if the editor beats live to a compilation using Elm, and it causes the cache corruption situation. Is a user even able to run `lamdera reset` in this context somehow?

Did you have any luck getting more information on what the error was when you added a `.settings` file?

**Comment by Dylanb-dev** [comment]
@supermario 

I think the flags being passed to `make` on the lamdera binary weren't getting passed in correctly, is there anything that could be causing lamdera make command to behave differently to elm binaries make?

**Comment by supermario** [comment]
@lolcookie which commands/flags were you trying exactly?

**Pull request closed without merging**